example/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  pull_to_refresh: ^1.6.3
+  pull_to_refresh: 1.6.3
```

Comment:
Apparently, the last version of pull to refresh generates somes issues during the build, so I used the previous one.
See this for more information: https://github.com/peng8350/flutter_pulltorefresh/issues/424